config/global.ini.php

```diff
@@ -833,6 +833,10 @@
 ; If set to 1, when rearchiving reports in the past we do not rearchive segment data with those reports. Default is 0.
 rearchive_reports_in_past_exclude_segments = 0
 
+; If set to 1, then social and search engine definitions files will be downloaded from Matomo remotely to update
```

Comment:
These are actually downloaded from github see https://github.com/matomo-org/matomo/blob/4.4.1/plugins/Referrers/Tasks.php#L33-L49 and not from Matomo. This is something we need to improve in the future to "proxy" or serve them through Matomo servers.

Reply:
@tsteur Thanks for that. I've made a change based on your suggestion :)